app/routes/pages/components/PageHierarchy.css

```diff
@@ -3,6 +3,7 @@
 .sidebar {
   display: flex;
   flex-direction: column;
+  margin: 0 -0.2em;
```

Comment:
Why the negative margin? Kind of hacky, but I'll allow it.

Reply:
Good catch, I've removed the negative margin and made the overall margin less!